site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,196 @@
++++
+date = "2020-06-07T03:15:00-03:00"
+title = "Progress Report May 2020"
+author = "GoldenX86"
+forum = 0
++++ 
+
+Hello yuzu users! How are you doing?
+
+In this episode of yuzu: Trials and Tribulations, we offer you: major rewrites, massive performance gains, 
+stability improvements, bug fixes and graphical corrections.
+More, after the break.
+<!--more-->
+
+## The worst kept secret
+
+It has its own [article](https://yuzu-emu.org/entry/yuzu-prometheus/), and it has been already guessed to hell and back before the official announcement.
+`Project Prometheus`, teased so much in yuzu’s official Discord server, is a proper multithreaded emulation of the 4 CPU cores the Nintendo Switch offers.
+This brings a massive performance boost to users with CPUs with 4 physical cores or more. But for this to happen, a lot of groundwork was needed.
+Besides changes previously discussed in the past reports, old external libraries (which yuzu needs to operate) needed to be updated,
+and with that, some changes were needed for our Linux users. 
+
+Thanks to [jroweboy’s](https://github.com/jroweboy) work, yuzu now uses [conan](https://conan.io/),
+helping the project manage dependencies, and letting Linux distributions use their native ones when possible. 
```

Comment:
Change `manage dependencies` to `[manage dependencies](https://github.com/yuzu-emu/yuzu/pull/3735)`